lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -2071,68 +2067,86 @@ defmodule Kernel.SpecialForms do
   exception by its name. All the following formats are valid patterns
   in `rescue` clauses:
 
-      # Rescue a single exception without binding the exception
-      # to a variable
-      try do
-        UndefinedModule.undefined_function
-      rescue
-        UndefinedFunctionError -> nil
-      end
+  Rescue a single exception without binding the exception to a variable:
 
-      # Rescue any of the given exception without binding
-      try do
-        UndefinedModule.undefined_function
-      rescue
-        [UndefinedFunctionError, ArgumentError] -> nil
-      end
+      iex> try do
+      ...>   1 / 0
+      ...> rescue
+      ...>   ArithmeticError -> :rescued
+      ...> end
+      :rescued
 
-      # Rescue and bind the exception to the variable "x"
-      try do
-        UndefinedModule.undefined_function
-      rescue
-        x in [UndefinedFunctionError] -> nil
-      end
+  Rescue any of the given exception without binding:
+
+      iex> try do
+      ...>   1 / 0
+      ...> rescue
+      ...>   [ArithmeticError, ArgumentError] -> :rescued
+      ...> end
+      :rescued
+
+  Rescue and bind the exception to the variable `x`:
+
+      iex> try do
+      ...>   1 / 0
+      ...> rescue
+      ...>   x in [ArithmeticError] -> [:rescued, is_exception(x)]
+      ...> end
+      [:rescued, true]
+
+  Rescue different errors with separate clauses:
+
+      iex> try do
+      ...>   1 / 0
+      ...> rescue
+      ...>   ArgumentError -> :rescued_argument_error
+      ...>   ArithmeticError -> :rescued_arithmetic_error
+      ...> end
+      :rescued_arithmetic_error
+
+  Rescue all kinds of exceptions and bind the rescued exception
+  to the variable `x`:
+
+      iex> try do
+      ...>   1 / 0
+      ...> rescue
+      ...>   x in [ArithmeticError] -> [:rescued, is_exception(x)]
```

Comment:
I think "Rescue all kinds of exception" used to be just
```suggestion
      ...>   x -> [:rescued, is_exception(x)]
```